mubench.reviewsite/templates/review.phtml

```diff
@@ -335,6 +397,18 @@ use MuBench\ReviewSite\Model\Experiment;
                     }
                     b.type = "hidden";
                 }
+                function onInput() {
+                    var tag_name_input = document.getElementById("tag-input").value;
+                    var suggested_tags = document.getElementById("tags").childNodes;
+                    var form = document.getElementById("tag-form");
+                    // loops over tags in the autocompletion list to submit the form when one is clicked
```

Comment:
Since this is triggered when the input changes, can I ever insert a tag `blafoo` when a tag `bla` exists? To my understanding, this listener will submit the form once I type `bla`. Isn't there a way to register an onClick listener on the datalist options or something?